tracer/src/Datadog.Trace/Configuration/IntegrationId.cs

```diff
@@ -60,6 +60,7 @@ internal enum IntegrationId
         OpenTelemetry,
         PathTraversal,
         Ldap,
-        Ssrf
+        Ssrf,
+        AzureServiceBus,
```

Comment:
FYI Creating a new `IntegrationId` creates new settings, you need to make sure to update the telemetry intake block list 🙂 

(There are also unit tests for this)

Reply:
Ah thanks for the reminder, done in 45ef28044932456c6989af0f9228e47df207cc31